src/charm.py

```diff
@@ -63,10 +63,9 @@ def __init__(self, *args):
 
     def _on_install(self, _: ops.InstallEvent):
         """Handle install event."""
-        if self.workload.install():
-            self.unit.set_workload_version(self.workload.get_version())
-        else:
+        if not self.workload.install():
             self._set_status(Status.SNAP_NOT_INSTALLED)
+        # self.unit.set_workload_version(self.workload.get_version())
```

Comment:
**question** why are we disabling this?

Reply:
Oh, I see. I tried installing the snap locally to play around with this. I have been having some issues with the python libs imports. I believe we can workaround these, using

```
PYTHONPATH=.... /snap/charmed-karapace/current/bin/karapace --version
```

But honestly, I believe it is just much better if we just add another app to the snap such that we can simply do:

```
charmed-karapace.karapace --version
```

This way people could also start the karapace service by themselves, not being attached to a deamon

Reply:
I agree. I'll open a PR on the snap for this